3rdparty/openblas/openblas.cmake

```diff
@@ -8,7 +8,7 @@ else()
     set(OPENBLAS_TARGET "NEHALEM")
```

Comment:
[nit/future] We should sync this with the ISPC arch. Also, is there a way to set this to the arch of the current machine? That will likely give a boost in benchmarks.